vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -489,6 +489,7 @@ export default class SbcHeader extends Mixins(NavigationMixin) {
       // checking for account status
       await this.checkAccountStatus()
     }
+    this.$emit('account-data-loaded')
```

Comment:
How do you see this being used? Would it update a store property that a UI could do something with (or await)?